pylab/website/forms.py

```diff
@@ -9,15 +9,23 @@ class ProjectForm(forms.ModelForm):
 
     class Meta:
         model = Project
-        fields = ('title', 'description')
+        fields = ('title', 'description', 'url', 'project_status', 'project_status_description')
         widgets = {
             'description': forms.Textarea(attrs={'rows': 16}),
+            'project_status_description': forms.Textarea(attrs={'rows': 6}),
```

Comment:
We don't need these fields in new project form. Leave this for as it is, we will set project status, status description and implementation url in the Django admin.